src/alphabase/base64url.cljc

```diff
@@ -0,0 +1,30 @@
+(ns alphabase.base64url
+  "Base 64 optimized namespace"
+  (:require [clojure.set :as set]
+            #?@(:cljs [[goog.crypt.base64 :as g-base64]]))
+  #?(:clj (:import (java.util Base64))))
+
+#?(:clj (set! *warn-on-reflection* true))
+
+(def ^:const base64url-chars "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.")
+(def ^:const base64url-set (set base64url-chars))
+
+(defn encode
+  "Converts a byte array into a base65URL string."
```

Comment:
s/base65/base64/ :)

Reply:
:facepalm: I'm just going to push this fix straight to master